modules/auxiliary/gather/vmware_vcenter_vmdir.rb

```diff
@@ -0,0 +1,172 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+# I'm tired of reinventing the wheel, so use a gem this time
+require 'net-ldap'
```

Comment:
Would it make more sense to create a LDAP library stub which simply loads this gem and defines a few `datastore` options?

Then you wouldn't need to do this:

```ruby
      Opt::RHOST,      # XXX: No included mixin provides this
      Opt::RPORT(389), # XXX: No included mixin provides this
```

Reply:
Yep, and we can incorporate much of the functionality in this module.

Reply:
But for now, I'll make the wrapper.